### PR TITLE
fix: use Be.Vlaanderen.Basisregisters.Sqs

### PR DESCRIPTION
**Pull request by lvermeulen**

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -2,194 +2,52 @@
 version 7.0.2
 framework: net6.0
 source https://api.nuget.org/v3/index.json
-nuget Be.Vlaanderen.Basisregisters.Build.Pipeline 6.0.5 //"
+nuget Be.Vlaanderen.Basisregisters.Build.Pipeline 6.0.3 //"
```
Comment (ArneD) [inline build.fsx:5]: why is this file changed?

Comment (lvermeulen) [reply build.fsx:5]: manual change - bad?
